usaspending_api/api_contracts/contracts/v2/agency/toptier_code/def_code.md

```diff
@@ -0,0 +1,104 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# List Disaster Emergency Fund Codes [/api/v2/agency/{toptier_code}/def_code/{?fiscal_year,filter,order,sort,page,limit}]
+
+Returns a list of Disaster Emergency Fund Codes in the Agency's appropriations for a single fiscal year
+
+## GET
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "number"
+            }
+    + Parameters
+        + `toptier_code`: 086 (required, number)
+            The toptier code of an agency (could be a CGAC or FREC) so only numeric character strings of length 3-4 are accepted.
+        + `fiscal_year` (optional, number)
+            The desired appropriations fiscal year. Defaults to the current FY.
+        + `filter` (optional, string)
```

Comment:
If this throws off our use of `filter` in lots of other places for this purpose, maybe disregard, but I think `keyword` or `term` or something like that is more appropriate for what this is being used for. 

"Filter" is a more general thing that could apply to many facets. And in fact we use it in that way in `POST` request bodies to be a filter object _with_ many facets.

Reply:
>Does this alter an existing contract?
It does alter an existing endpoint. The endpoint has been released in prod, but is not yet used by the website (Agency Profile V2 page). I don't mind changing the fields back if it's okay with the frontend. 

>If this throws off our use of filter in lots of other places for this purpose, maybe disregard, but I think keyword or term or something like that is more appropriate for what this is being used for.
I agree with this too, but `filter` is already already being in these existing endpoints, so we'd have to go with a "bump the version" option if we wanted to rename this field

Reply:
I revered the obligation/outlay field names and left the filter parameter as is to prevent a new version from being necessary